config/crd/bases/nfd.kubernetes.io_nodefeaturediscoveries.yaml

```diff
@@ -5,7 +5,6 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.4.1
-    api-approved.kubernetes.io: "unapproved, experimental-only"
```

Comment:
Is this desired?

Reply:
yes, is not needed any more, we can now host the default yaml we get from `make manifest` and count on `kustomize` to apply annotation when `make deploy`